cache/fast_lru_cache.h

```diff
@@ -365,6 +368,8 @@ class ALIGN_AS(CACHE_LINE_SIZE) LRUCacheShard final : public CacheShard {
 
  private:
   friend class LRUCache;
+  friend class ROCKSDB_NAMESPACE::fast_lru_cache::FastLRUCacheTest;
```

Comment:
ROCKSDB_NAMESPACE::fast_lru_cache:: should not be needed here.

Reply:
You are right.